queries/highlights.scm

```diff
@@ -1,76 +1,104 @@
-; Comments
-(line_comment) @comment
+(comment) @comment
 
-(block_comment) @comment
+(ty (id)) @type
+
+(valid_semver) @property
+
+(world_item
+  name: (id) @property)
+
+(interface_item
+  name: (id) @property)
+
+(type_item
+  alias: (id) @property)
+
+(func_item
+  name: (id) @function)
+
+(handle
+  (id) @type)
+
+(named_type
+  name: (id) @variable.parameter)
+
+(record_item
+  name: (id) @variable)
+
+(record_field
+  name: (id) @variable)
+
+(flags_items
+  name: (id) @variable)
+
+(flags_body
+  (id) @property)
+
+(variant_items
+  name: (id) @variable)
+
+(variant_case
+    name: (id) @variable)
+
+(enum_items
+  name: (id) @variable)
+
+(enum_body
+  enum_cases: (id) @property)
+
+(resource_method
+  "constructor" @constructor)
+
+[
```

Comment:
To comply with [nvim-treesitter](https://github.com/nvim-treesitter/nvim-treesitter/blob/master/CONTRIBUTING.md#keywords) standards, please distinguish between different keyword variants, such as `@keyword.import` (import, export etc.) and `@keyword.type` (record, variant etc)

Reply:
should  be better now